changelog/LATEST.md

```diff
@@ -15,7 +15,7 @@ View all [changelogs](https://github.com/Divine-Journey-2/Divine-Journey-2/tree/
 
 ## QoL Improvements:
 
-
+- Added the Lizard, Druden, Saguaro Worm and Cyclops to the Roots Ritual: Summon Creatures.
```

Comment:
definitely belongs in Balance Adjustments. include serial comma (Worm, and Cyclops)